src/Components/drills/FrisbeeDrillIllustration.js

```diff
@@ -79,12 +80,25 @@ const FrisbeeDrillIllustration = props => {
     );
   };
 
-  const displayAnimation = ({ illustrationSource, instruction }) => {
+  const displayVideo = videoIllustration => {
+    if (typeof videoIllustration !== 'undefined') {
+      return (
+        <View style={styles.centerContainer}>
+          <TouchableOpacity onPress={() => navigation.navigate('VideoPage', { videoIllustration })}>
+            <MaterialCommunityIcons name="video" color={theme.COLOR_PRIMARY} size={35} />
+          </TouchableOpacity>
+        </View>
+      );
+    }
+  };
+
+  const displayAnimation = ({ illustrationSource, instruction, videoIllustration }) => {
     return (
       <>
         <View style={styles.contentWrapper}>
           <Animation widthRatio={1} heightRatio={1 / 2} animation={new Drill(illustrationSource)} />
         </View>
+        <View>{displayVideo(videoIllustration)}</View>
```

Comment:
Why don't you use a step with a video to keep it within the carousel?

It seems to me that we are breaking here the paradigm of a drill being made of steps and moving against KISS principle (https://en.wikipedia.org/wiki/KISS_principle)

Reply:
The carroussel is slowing down everything and Here is the video to illustrate one step. It could look as another step if we have to swipe to see it.